src/config.h

```diff
@@ -52,11 +52,11 @@ struct x509_digest {
 #define COOKIE_SIZE	300
 
 struct vpn_config {
-	char 		gateway_host[FIELD_SIZE];
+	char 		gateway_host[FIELD_SIZE + 1];
 	struct in_addr	gateway_ip;
 	uint16_t	gateway_port;
-	char		username[FIELD_SIZE];
-	char		password[FIELD_SIZE];
+	char		username[FIELD_SIZE + 1];
+	char		password[FIELD_SIZE + 1];
```

Comment:
I wanted mainly to unify those fields with cookie's policy: maybe what really needs to be changed it the cookie.